cupy/__init__.py

```diff
@@ -1301,6 +1305,9 @@ def _should_use_rop(self, a):
 equal = logic.comparison.equal
 not_equal = logic.comparison.not_equal
 
+all = logic.truth.all
+any = logic.truth.any
```

Comment:
It affects this line: https://github.com/pfnet/chainer/pull/493/files#diff-944d22b7359c840a90125a0eaca62545R1447